intro.md

```diff
@@ -35,6 +38,11 @@ As I said, i want to develop great real-social applications. Applications that c
 
 One well explored, and popular use case is Hospitality Exchange. It's pretty clear what the system should do.
 
+[comment]: <> (would be great if you mentioned somewhere here, here-here or elswhere on this page, how your collaboration with me or/and OHN started, what OHN is about and how it fits together with Your agenda. Maybe also a few words about sleepy.bike, who is it for and where it comes from? I'll be happy to discuss or draft parts of this if it was helpful.)
```

Comment:
Great. Definitely, let's include.

If you have some text, or could draft some, that could be helpful. We could even make a shorter comment here and link it to a longer text with its own separate page. :)